### PR TITLE
Improve credit card flow

### LINKED ISSUE #1654: Update Product plans page link in select plan
After creation of plans pricing and description page,
We should add the link to the Change plan modal in Subscription settings

It's in `ROUTE_LINKS.ProductPlans` from `packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/SelectPlan.tsx`

### LINKED ISSUE #1825: Improve credit card flow design
From https://github.com/ChainSafe/ui-monorepo/pull/1804#issuecomment-989078595

1. For the first modal it appears quite text-dense from the image, but if we **stick to lighter colors for copy that isn't as important** then it would help combat that (sample below). In addition, the **bottom-left link has been updated** because I've confirmed that we don't actually have a page that goes into detail about our plans (most likely a V2 thing).

image

2. Use the modal cross at the top right to close the mobile completely instead of a "Cancel" button, and remove any "Cancel" button. We should have a "back" button on most steps to be able to go to the previous step.
image

3. We should remove any breadcrumb

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #1825 
closes #1654  

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

**Review by asnaith — APPROVED** [review]

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx [hunk 1]
```diff
@@ -78,6 +86,11 @@ const ChangeProductModal = ({ onClose }: IChangeProductModal) => {
     }
   }
 
+  const closeIcon = <CrossIcon
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx]: The modal has a built in close icon, Why not use this one?

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ConfirmPlan.tsx [hunk 2]
```diff
@@ -85,14 +86,22 @@ const useStyles = makeStyles(({ constants, palette }: CSFTheme) =>
     error: {
       marginTop: constants.generalUnit,
       color: palette.error.main
+    },
+    crossIcon: {
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ConfirmPlan.tsx]: Same here, this is duplicated isn't it?

**Review by Tbaut — CHANGES_REQUESTED** [review]

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx (hunk 1 above)
Comment (tanmoyAtb) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx]: Yes, this is duplicated, 
I tried using the the default icon, But each page in this modal has different sizes and the cross icon was getting outside the modal screens, I tried making it work, but it was getting to too much refactoring.

Comment (Tbaut) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx]: Hmm, If there's something broken elsewhere we should 100% fix it, eventually in another PR, but doing what we're doing here doesn't feel right. Happy to help there if you want.

Comment (tanmoyAtb) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx]: Alright, So managed to get the cross icon to work within the modal component !

Comment (Tbaut) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx]: yes yes yes awesome! :raised_hands: checking it out

**Review by Tbaut — APPROVED** [review]
:100: tested mobile view on many modals, and looks great

**Pull request merged**